account_invoice_user_authorization_payment/views/account_move_views.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+
+    <record id="user_authorization_payment_tree" model="ir.ui.view">
+        <field name="name">account.move.tree (User Authorization Payment)</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_in_invoice_tree"/>
+        <field name="groups_id" eval="[(4, ref('account.group_account_user'))]"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='name']" position="after">
+                <field name="create_uid" />
+            </xpath>
+        </field>
+    </record>
+
+    <record id="user_authorization_payment_form" model="ir.ui.view">
+        <field name="name">account.move.form (User Authorization Payment)</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form"/>
+        <field name="groups_id" eval="[(4, ref('account.group_account_user'))]"/>
+        <field name="arch" type="xml">
+            <xpath expr="//group//field[@name='partner_bank_id']" position="after">
+                <field name="user_authorization_payment" />
```

Comment:
A invisible attribute that hides this field for those account moves not corresponding to purchase/vendor invoices should be needed